source-academy/frontend
# Grading: Viewable Assessment Briefings

### Description



We've added a tab to view the briefings (long summary) inside the assessment for grading that resolves [#2310](https://github.com/source-academy/frontend/issues/2310)

### Note
- This pull request requires this pull request in the backend to be compatible - https://github.com/source-academy/backend/pull/1059 
- The implementation changes the redux state of ```grading``` due to a new query response format from the backend API.
- It may break code that uses the interfaces of ```GradingAnswers``` (previously known as ```Grading```) and ```GradingQuery```, but I have tested it as much as possible.

### Type of change


### How to test



- Install the required backend PR
- Open Source Academy and go to Grading
- Grade any of the assessments
- There should be a new tab on the UI that allows you to see the briefing (see screenshot below)

![Grading Briefing Preview](https://github.com/source-academy/frontend/assets/20984062/2fba127d-3266-4272-b98f-20b10f30d514)

### Checklist
